week-4/Homework/mandatory/2-writers.js

```diff
@@ -53,19 +53,36 @@ Exercise 1:
   insert corresponding values to the place holder that are indicated in courle braces:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
-
-
+console.log(
+  writers.map(
+    (p) =>
+      `Hi, my name is ${p.firstName}  ${p.lastName}. I am ${p.age} years old, and work as a ${p.occupation}.`
+  )
+);
 

```

Comment:
Good work, but you have a problem here
.map is used when you want to store data in new variable, if you don't want that you should use forEach.